hospital/src/main/resources/db.changelog/v1.0/10-create-doctors-table.sql

```diff
@@ -0,0 +1,18 @@
+create table doctor (
+    id INT not null,
+    primary key (id),
+    constraint iddoctor
+      foreign key (id)
+    references user (id)
+
+) engine=InnoDB
+
+NEXT_SCRIPT
```

Comment:
Do we need this custom NEXT_SCRIPT? The default char is `;` which seems suitable

Reply:
done